Mathlib/Algebra/Opposites.lean

```diff
@@ -360,12 +360,9 @@ theorem op_div [Div α] (a b : α) : op (a / b) = op a / op b :=
   rfl
 #align add_opposite.op_div AddOpposite.op_div
 
--- porting note: this lemma looked wrong to me -- is it wrong in mathlib3?
--- The types of a and b were α not αᵃᵒᵖ.
```

Comment:
fix has been backported to mathlib3, hence this comment is superflous

Reply:
Agreed!